webapp/client/src/components/FormFieldRadio.js

```diff
@@ -0,0 +1,132 @@
+import { useState } from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import FormFieldScaffolding from "./FormFieldScaffolding";
+import FieldLabelForSeparatedFields from "./FieldLabelForSeparatedFields";
+import { optionsPropType } from "../lib/propTypes";
+
+const Radio = styled.div`
+  input[type="radio"] {
+    opacity: 0;
+    position: absolute;
+  }
+
+  input[type="radio"] + label {
+    display: flex;
+    padding-left: 0;
+    padding-top: 0;
+    font-size: var(--text-medium);
+  }
+
+  input[type="radio"] + label::before {
+    content: "";
+    display: inline-block;
+    vertical-align: bottom;
+    width: 30px;
+    height: 30px;
+    min-width: 30px;
+    margin-right: 12px;
+    background: url("/icons/radio_button_default.svg") no-repeat center;
+  }
+
+  input[type="radio"]:checked + label::before {
+    background: url("/icons/radio_button_checked.svg") no-repeat center;
+  }
+
+  input[type="radio"]:checked + label:hover::before {
+    background: url("/icons/radio_button_checked_hover.svg") no-repeat center;
+  }
+
+  input[type="radio"]:not(:checked):focus + label::before {
+    background: url("/icons/radio_button_focus.svg") no-repeat center;
+  }
+
+  input[type="radio"]:checked:focus + label::before {
+    background: url("/icons/radio_button_checked_focus.svg") no-repeat center;
+  }
+
+  .radio-button-list {
+    padding-top: var(--spacing-02);
+  }
+`;
+
+function FormFieldRadio({
+  fieldName,
+  fieldId,
+  options,
+  value,
+  required,
+  autofocus,
+  label,
+  details,
+  errors,
+  onChangeHandler,
+}) {
+  const [selectedValue, setSelectedValue] = useState(value);
+
+  const toggleRadioButton = (event) => {
+    setSelectedValue(event.target.value);
+  };
+
+  return (
+    <FormFieldScaffolding
+      {...{
+        fieldName,
+        errors,
+      }}
+      hideLabel
```

Comment:
@frederike-ramin Another thing I noticed: in the jinja components the Radio Field was `hideLabel=True` _and_ `hideErrors=True`. Just wanted to check you considered this!

Reply:
Damn. You're making a good point. Thanks for pointing that out!